From 0a1f2e57cc29cd258eb8927139727c0a1a05e4e2 Mon Sep 17 00:00:00 2001
From: Phinex Hung <phinex@realtek.com>
Date: Tue, 20 Feb 2024 13:10:47 +0800
Subject: [PATCH] dts: realtek: rtd1619b: Enable SATA Port0

-SATA Port 0 and PCIe are shared SERDES, should disable PCIe1
---
 .../boot/dts/realtek/rtd1619b-bleedingedge.dtsi      | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/realtek/rtd1619b-bleedingedge.dtsi b/arch/arm64/boot/dts/realtek/rtd1619b-bleedingedge.dtsi
index b570fb3c77cc..e4ad623f64df 100644
--- a/arch/arm64/boot/dts/realtek/rtd1619b-bleedingedge.dtsi
+++ b/arch/arm64/boot/dts/realtek/rtd1619b-bleedingedge.dtsi
@@ -406,11 +406,11 @@ &nic {
 };
 
 &pcie1 {
-	status = "okay";
+	status = "disabled";
 };
 
 &pcie1_phy {
-	status = "okay";
+	status = "disabled";
 };
 
 &pcie2 {
@@ -580,7 +580,7 @@ &npu {
 };
 
 &sata_phy {
-	status = "disabled";
+	status = "okay";
 };
 
 &sata_phy0 {
@@ -588,7 +588,7 @@ &sata_phy0 {
 	/delete-property/ tx-swing;
 	tx-emphasis = <3>, <(-2)>, <(-3)>; /* tx emphasis adjust sample : <gen1>, <gen2>, <gen3> */
 	/delete-property/ tx-emphasis;
-	status = "disabled";
+	status = "okay";
 };
 
 &sata_phy1 {
@@ -596,11 +596,11 @@ &sata_phy1 {
 };
 
 &ahci_sata {
-	status = "disabled";
+	status = "okay";
 };
 
 &sata_port0 {
-	status = "disabled";
+	status = "okay";
 };
 
 &sata_port1 {
-- 
2.42.0

